packages/@sanity/base/src/presence/FieldPresence.tsx

```diff
@@ -31,7 +31,7 @@ function FieldPresenceWithOverlay(props: FieldPresenceProps) {
   const {presence = contextPresence, maxAvatars = DEFAULT_MAX_AVATARS_FIELDS} = props
   const ref = React.useRef(null)
 
-  useReporter(useId(), () => ({presence, element: ref.current!, maxAvatars: maxAvatars}))
+  useReporter(useId() || '', () => ({presence, element: ref.current!, maxAvatars: maxAvatars}))
```

Comment:
Why do we need the fallback to empty string here? If useId() can return a non-string I think it's better if we throw a descriptive error instead. The id's here needs to be unique, and this change is likely to yield an error further down the line that becomes hard to trace.

Reply:
Did this to make the TS type check to pass. Will try your suggestion 👍